### DISCUSSION TITLE
Using Markdown SSG with CSS-resetting in _app

### DISCUSSION THREAD

**Discussion by maxeth** [vercel/next.js#16759] [Help] [answered]
Hey,
I'm using ChakraUI for my hybrid application which required me to add the ` <CSSReset/>` Component to my `_app.js` file to delete all default browser CSS styles.

Now I wanted to extend my App by a blog that uses static site generation with `getStaticProps`  which get populated by markdown files. 

The problem: resetting the initial CSS with the  ` <CSSReset/>` component (which I need for the rest of my app) deletes all the markdown HTML formatting/styles. So my blog pages look like one single `<p>` tag because all the styles get removed.

Is there a way to NOT render that  ` <CSSReset/>` component inside my `_app.js` or to use the default `_app.js` when a specific path (the path of my static site) is being requested?

I don't want to hardcode every single blog-page HTML because that would make the app pretty much unmaintainable.
 
Thanks in advance

**Comment by leerob** [comment, marked as answer]
You have a couple of options here. 

1. You could use `useRouter` to check the current path, and if it's not `/whatever-route` you want, then don't render `<CSSReset />`
2. Style your markdown components! You can use `MDXProvider ` to forward Chakra UI components. [Here's an example](https://github.com/leerob/leerob.io/blob/master/pages/_app.js#L68)

**Reply by maxeth** [reply]
Option one was a great solution for me, thanks a lot!
